app/src/test/kotlin/compiler/intermediate_form/FunctionDetailsGeneratorTest.kt

```diff
@@ -0,0 +1,353 @@
+package compiler.intermediate_form
```

Comment:
I don't analyze this file in this review as some aspects of FDG class are to be discussed/corrected.